docs/modules/ROOT/pages/api/account.adoc

```diff
@@ -0,0 +1,234 @@
+:github-icon: pass:[<svg class="icon"><use href="#github-icon"/></svg>]
+:snip6: https://github.com/ericnordelo/SNIPs/blob/feat/standard-account/SNIPS/snip-6.md[SNIP-6]
+
+= Account
+
+Reference of interfaces, presets, and utilities related to account contracts.
+
+== Core
+
+[.contract]
+[[ISRC6]]
+=== `++ISRC6++` link:https://github.com/OpenZeppelin/cairo-contracts/blob/cairo-2/src/account/interface.cairo#L12[{github-icon},role=heading-link]
+
+[.hljs-theme-light.nopadding]
+```javascript
+use openzeppelin::account::interface::ISRC6;
+```
+
+Interface of the SRC6 Standard Account as defined in the {snip6}.
+
+[.contract-index]
+.Functions
+--
+* xref:#ISRC6-\\__execute__[`++__execute__(calls)++`]
+* xref:#ISRC6-\\__validate__[`++__validate__(calls)++`]
+* xref:#ISRC6-is_valid_signature[`++is_valid_signature(hash, signature)++`]
+--
+
+[#ISRC6-Functions]
+==== Functions
+
+[.contract-item]
+[[ISRC6-__execute__]]
+==== `[.contract-item-name]#++__execute__++#++(calls: Array<Call>) → Array<Span<felt252>>++` [.item-kind]#external#
+
+Executes the list of calls as a transaction after validation.
+
+Returns an array with each call's output.
+
+NOTE: The `Call` struct is defined in https://github.com/starkware-libs/cairo/blob/main/corelib/src/starknet/account.cairo#L3[corelib].
+
+[.contract-item]
+[[ISRC6-__validate__]]
+==== `[.contract-item-name]#++__validate__++#++(calls: Array<Call>) → felt252++` [.item-kind]#external#
+
+Validates a transaction before execution.
+
+Returns the short string `'VALID'` if valid, otherwise ir reverts.
+
+[.contract-item]
+[[ISRC6-is_valid_signature]]
+==== `[.contract-item-name]#++is_valid_signature++#++(hash: felt252, signature: Array<felt252>) → felt252++` [.item-kind]#external#
+
+Validates whether a signature is valid or not for the given message hash.
+
+Returns the short string `'VALID'` if valid, otherwise ir reverts.
+
+[.contract]
+[[Account]]
+=== `++Account++` link:https://github.com/OpenZeppelin/cairo-contracts/blob/cairo-2/src/account/account.cairo#L27[{github-icon},role=heading-link]
+
+:OwnerAdded: xref:Account-OwnerAdded[OwnerAdded]
+:OwnerRemoved: xref:Account-OwnerRemoved[OwnerRemoved]
+
+[.hljs-theme-light.nopadding]
+```javascript
+use openzeppelin::account::Account;
+```
+Account contract implementation extending xref:ISRC6[`ISRC6`].
+
+[.contract-index]
+.Utilities
+--
+* xref:#Account-assert_only_self[`++InternalImpl::assert_only_self(self)++`]
```

Comment:
Should we document `_execute_calls` and `_execute_single_call`?

Reply:
i'm not sure that's part of the API we offer, up to debate though

Reply:
Agreed, they're more helpers than part of the API per se. I lean toward documenting them because whether we mean for them to be part of the API or not, they're accessible. No strong opinions though

Reply:
I'm leaning towards letting them out of the docs, but changing the attribute from `#[internal]` to `#[private]`. This won't make the functions inaccessible, but I think people will understand private stuff not being part of the documented API (or we can explicitly say it in our codying style doc).

ex:

```
    #[private]
    fn _execute_single_call(call: Call) -> Span<felt252> {
        let Call{to, selector, calldata } = call;
        starknet::call_contract_syscall(to, selector, calldata.span()).unwrap()
    }
```

Reply:
Nice and simple! I think this looks good